rtic-monotonics/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "rtic-monotonics"
+version = "0.1.0"
```

Comment:
Both rtic-channel and rtic-monotonics start at 1.0.0 currently

Should we maybe do "v1.0.0-alpha.0" for all of these, and then when RTIC v2 reaches stable these hopefully also reached stable v1?

Reply:
SGTM

Reply:
Is it really needed, nobody will rely on them being stable in any case?